TestResultSummaryService/parsers/Test.js

```diff
@@ -1,32 +1,58 @@
 const Parser = require( './Parser' );
-const regex = /.*?===============================================\r?\n.*?Running test (.*?) \.\.\.\r?\n.*?===============================================\r?\n.*?([\S\s]*?\1 Start Time\: .* Epoch Time \(ms\)\: (.*)[\s\S]*?\1_(.*?)\r?\n[\S\s]*?\1 Finish Time\: .* Epoch Time \(ms\)\: (.*))\r?\n/g;
+const regexRunningTest = /.*?Running test (.*?) \.\.\.\r?/;
+const regexFinishTime = /.*?Finish Time\: .* Epoch Time \(ms\)\: (.*).*/;
+const regexStartTime = /.*?Start Time\: .* Epoch Time \(ms\)\: (.*).*/;
+var results = [];

```

Comment:
- `const results = [];` not `var`. We should try to use ES6 syntax.
- we should move `const results = [];` inside of extract()